packages/react-native-reanimated/jest-setup.js

```diff
@@ -1,2 +1,19 @@
 delete global.MessageChannel;
 require('./src/jestUtils').setUpTests();
+
+global.__loggerConfig = {
+  logFunction: (data) => {
+    switch (data.level) {
+      case 'warn':
+        console.warn(data.message.content);
+        break;
+      case 'error':
+      case 'fatal':
+      case 'syntax':
+        console.error(data.message.content);
+        break;
+    }
+  },
+  level: 'warn',
+  strict: false,
+}
```

Comment:
is it really necessary?

Reply:
If the `__loggerConfig` is global, yes it is necessary. `__loggerConfig` is not initialized in jest as the file in which we register `__loggerConfig` in globals is not executed.

When I change `__loggerConfig` to use import/export way as I made with `ReanimatedError` it should no longer be necessary.

Reply:
Is it still necessary? 😅